doxygen/dox/sw_changes_1.10.dox

```diff
@@ -0,0 +1,660 @@
+/** \page rel_spec_110_change Release Software Changes for HDF5 1.10
+
+\section sec_rel_spec_110_change Software Changes from Release to Release in HDF5 1.10
+For a description of the major new features that were introduced,
+please see \ref sec_rel_spec_110_feat.
+
+This page provides information on the changes that a maintenance developer needs to be
+aware of between successive releases of HDF5, such as:
+\li New or changed features or tools
+\li Syntax and behavioral changes in the existing application programming interface (API)
+\li Certain types of changes in configuration or build processes
+
+Note that bug fixes and performance enhancements in the C library are automatically picked
+up by the C++, Fortran, and Java libraries.
+
+The following information is included below.
+\li \ref subsec_rel_spec_110_change_compatibility
+\li \ref subsec_rel_spec_110_change_9versus8
+\li \ref subsec_rel_spec_110_change_8versus7
+\li \ref subsec_rel_spec_110_change_7versus6
+\li \ref subsec_rel_spec_110_change_6versus5
+\li \ref subsec_rel_spec_110_change_5versus4
+\li \ref subsec_rel_spec_110_change_4versus3
+\li \ref subsec_rel_spec_110_change_3versus2
+\li \ref subsec_rel_spec_110_change_2versus1
+\li \ref subsec_rel_spec_110_change_1versus0
+\li \ref subsec_rel_spec_110_change_0versus8_16
+
+See [API Compatibility Reports for 1.10]() for information regarding compatibility with previous releases.
```

Comment:
We don't have this page any more.  Line 29 should be removed.

Reply:
I am going to resolve the compat reports in a separate PR. Unless some one has a major issue with it. Because we need to make sure the support site is properly updated and doxygen can access the files.

Reply:
Resolved the 1.14 with an update